### ISSUE TITLE
Want to join forces?

### ISSUE THREAD

**Issue by podusowski** [SaadiSave/horizons-rs#1]
Hi!
was looking for accessing the Horizons by myself and since I couldn't find anything working I just started writing it.

https://github.com/podusowski/rhorizons

I was wondering whether you'd be interested in joining forces or freeing crates.io `horizons` name?

**Comment by SaadiSave** [comment]
Yes, I'd like that. I'll discuss this in length tomorrow. I also have a bunch of uncommitted code that I will push to this repo soon.

**Comment by SaadiSave** [comment]
@podusowski What are your strengths? e.g. Parsers, Networks, etc.

**Comment by SaadiSave** [comment]
$B_4$ is $I_4$ and $B_2$ is $I_2$
